HeavyIonsAnalysis/EventAnalysis/src/HiEvtAnalyzer.cc

```diff
@@ -170,6 +176,12 @@ HiEvtAnalyzer::~HiEvtAnalyzer()
 void
 HiEvtAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
 {
+
+  //cleanup previous event
+  npus.clear();
+  tnpus.clear();
```

Comment:
Yes, I think so. I couldn't find anywhere a place where they were cleared. So I fixed that on my way as well